### ISSUE TITLE
error in userparameter_nginx.conf file

### ISSUE THREAD

**Issue by NMamont** [sfuerte/zbx-nginx#1]
here 
UserParameter=nginx.writting[*],/etc/zabbix/zabbix_agentd.scripts/nginx-stat.py -h $1 -p $2 -a writting

you have a double "t" in the word "writing" and this causes an error "ZBX_NOTSUPPORTED: Unsupported item key".
and your scripts require python> = 2.7.9, that mean  debian support with version 8 or higher, please specify this

and in your zabbix_template is specified {NGINX.HOST}, in zabbix 3.4 - zabbix interprets this as {HOST.IP} and sends a query of the type "nginx.accepted [{HOST.IP}, 80]", and in response "ZBX_NOTSUPPORTED: Special characters "\, ', ", `, *, ?, [, ], {, }, ~, $, !, &, ;, (, ), <, >, |, #, @, 0x0a" are not allowed in the parameters."
if we change it in the zabbix_template {NGINX.HOST}  to {HOST.IP}, the interpretation is correct, but we have an error of 403, because in nginx default conf allow - only for 127.0.0.1.
You must add it to allow host.ip in default.conf or change {NGINX.HOST} to 127.0.0.1

**Comment by sfuerte** [comment]
@NMamont, thanks for the follow up!
Fixed the typo, as well as added Python >= 2.7.9 and update Nginx config in the README. 

As for the `{NGINX.HOST}`, it's a bit more complicated.
The template (XML file, it was someone's template that we used and extended) defines that macro as:
```
                <macro>
                    <macro>{$NGINX_HOST}</macro>
                    <value>{HOST.IP}</value>
                </macro>
```

Unfortunately, Zabbix doesn't support nested macros yet. I'm planning to send a feature request to them for that. 
Depending on your environment, there are two workarounds at the moment: 
1) before importing XML template, delete the **template** macro definition and replace `{$NGINX_HOST}` with `{HOST.IP}`
2) re-define `{$NGINX_HOST}` macro for each host. We use this option at the moment. A bit inconvenient but gives you more flexibility as on some hosts Nginx can listen on localhost only or 1 out of 5 IP addresses, etc.
